RealmModelGenerator/AttributeDetailView.swift

```diff
@@ -45,28 +46,28 @@ class AttributeDetailView: NibDesignableView, NSTextFieldDelegate {
     }
     
     @IBInspectable var isIgnored:Bool {
-        set { self.ignoredCheckBox.state = newValue ? 1 : 0 }
-        get { return self.ignoredCheckBox.state == 1 ? true : false }
+        set { self.ignoredCheckBox.state = newValue ? NSControl.StateValue.on : NSControl.StateValue.off }
+        get { return self.ignoredCheckBox.state == NSControl.StateValue.on ? true : false }
     }
     
     @IBInspectable var isIndexed:Bool {
-        set { self.indexedCheckBox.state = newValue ? 1 : 0 }
-        get { return self.indexedCheckBox.state == 1 ? true : false }
+        set { self.indexedCheckBox.state = newValue ? NSControl.StateValue.on : NSControl.StateValue.off }
+        get { return self.indexedCheckBox.state == NSControl.StateValue.on ? true : false }
```

Comment:
use shorthand `.on` and `.off` where applicable.

Reply:
Done